site/assets/scss/_clipboard-js.scss

```diff
@@ -2,7 +2,8 @@
 //
 // JS-based `Copy` buttons for code snippets.
 
-.bd-clipboard {
+.bd-clipboard,
+.bd-edit {
```

Comment:
This whole duplication leads me to think we should use one common class for both buttons.

On the JS side we shouldn't reinvent the wheel since we have code that handles the clipboard tooltip.

Reply:
Agree on it. I'll look into using a single class only